modules/aws/state_manager/README.md

```diff
@@ -0,0 +1,40 @@
+## Terraform State Management Bucket
+
+This Terraform module will produce a State Management bucket that will allow you to manage the state of Terraform on an 
+environment for a specific project allowing you to manage what's on an environment on the environment itself instead of 
+locally.
```

Comment:
@nitpicking (not blocking): This sentence is a bit hard to read.

@suggestion:  change this to "This Terraform module will produce a State Management bucket that will allow you to store and manage the Terraform state for a specific project on a remote environment instead of locally.